src/Microsoft.Framework.Runtime.Hosting/Json/JsonDeserializerResource.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Runtime.Json
+{
+    internal class JsonDeserializerResource
+    {
+        internal static string Format_IllegalTrailingCharacterAfterLiteral(int value, string literal)
+        {
+            return string.Format("Illegal character {0} ({1:X4}) trails the literal name {2}", (char)value, value, literal);
+        }
+
+        internal static string Format_UnrecognizedLiteral(string literal)
+        {
+            return string.Format("Unrecognized json literal. {0} is expected.", literal);
```

Comment:
`json` --> `JSON` and please check this in all resources / exceptions / doc comments in the PR